.github/workflows/test-deploy.yml

```diff
@@ -22,10 +22,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.sha || github.ref }}
-      - uses: actions/setup-node@v2
-      - name: check lock
-        run: |
-          test $(cat package-lock.json | jq .lockfileVersion) == "2"
+      - run: jq --exit-status .lockfileVersion==2 < package-lock.json
```

Comment:
can we name it for verboseness?

Reply:
I usually avoid names when the command intent is “evident” but sure

Reply:
lol well there is no guessing what its for now 🙃 ❤️

Reply:
Anyway, pinging @casperdcl for opinions on terse pipeline syntax. (Help!) 😄

Reply:
Resolved with https://github.com/iterative/cml/pull/1082#discussion_r910301999